AAP-13268-EDA updated eda controller user guide (#1495)
ansible/aap-docs: downstream/modules/eda/con-system-level-monitoring.adoc

[id="system-level-monitoring"]

=  System level monitoring for {EDAcontroller}

[role="_abstract"]

> After characterizing your workload to determine how many rulebook activations you are running in parallel and how many events you are receiving at any given point, you must consider monitoring your {EDAcontroller}  host at the system level. Using system level monitoring to review information about {EDAName}’s performance over time helps when diagnosing problems or when capacity planning for future growth.

Reviewer: ```suggestion
After characterizing your workload to determine how many rulebook activations you are running in parallel and how many events you are receiving at any given point, you must consider monitoring your {EDAcontroller}  host at the system level. Using system level monitoring to review information about {EDAName}’s performance over time helps when diagnosing problems or when considering capacity for future growth.
```
Just a suggestion - this might be easier for auto-translate to parse

System level monitoring includes the following information:

* Disk I/O
* RAM utilization
* CPU utilization
* Network traffic 

> Higher CPU, RAM, or Disk utilization can affect the overall performance of {EDAcontroller}. For example, a high utilization of any of these system level resources indicates that either the {EDAcontroller} is running too many rulebook activations, or some of the individual rulebook activations are using a high volume of resources. In this case, it is necessary to increase your system level resources in order to support your workload.

Reviewer: ```suggestion
Higher CPU, RAM, or Disk utilization can affect the overall performance of {EDAcontroller}. For example, a high utilization of any of these system level resources indicates that either the {EDAcontroller} is running too many rulebook activations, or some of the individual rulebook activations are using a high volume of resources. In this case, you must increase your system level resources to support your workload.
```
